theseus/geometry/point_types.py

```diff
@@ -161,3 +161,9 @@ def exp_map(tangent_vector: torch.Tensor) -> "Point3":
     # added to avoid casting downstream
     def copy(self, new_name: Optional[str] = None) -> "Point3":
         return cast(Point3, super().copy(new_name=new_name))
+
+
+rand_point2 = Point2.rand
```

Comment:
We need to add these to `__init__.py` all the way to the top level. A good example to follow here are the group [operations](https://github.com/facebookresearch/theseus/blob/main/theseus/__init__.py#L25).